test/Microsoft.Azure.SignalR.Tests.Common/E2ETest/ITestServer.cs

```diff
@@ -1,14 +1,15 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Collections.Generic;
 using System.Threading.Tasks;
 using Microsoft.Extensions.Logging;
 
 namespace Microsoft.Azure.SignalR.Tests.Common
 {
     public interface ITestServer
     {
-        Task<string> StartAsync();
+        Task<string> StartAsync(Dictionary<string, string> configuration = null);
```

Comment:
Could you make the `HubConnectionManager` in the interface instead of casting, and also update https://github.com/Azure/azure-signalr/blob/dev/test/Microsoft.Azure.SignalR.Tests.Common/E2ETest/ServiceE2EFactsBase.cs which is also related to test server?

Reply:
Add to interface is fine, but scenarios in `ServiceE2EFactsBase` is more complicated. We need to use another pr to update it.

Reply:
ok